inference-engine/tests/ngraph_helpers/lpt_ngraph_functions/src/transformations_after_split_function.cpp

```diff
@@ -17,7 +17,7 @@ namespace builder {
 namespace subgraph {
 
 std::shared_ptr<Function> TransformationsAfterSplitFunction::get(const std::string transformationName) {
-    const auto input = std::make_shared<opset1::Parameter>(element::u8, Shape{ 1, 3, 16, 16 });
```

Comment:
The same fix as above. Because the block size is 3 for depth to space.

https://github.com/openvinotoolkit/openvino/blob/ca0cdb896f7a4d7019422897690b7c81cc508a75/inference-engine/tests/ngraph_helpers/lpt_ngraph_functions/src/transformations_after_split_function.cpp#L93